genColors.ts

```diff
@@ -35,6 +35,8 @@ const Colors = {
   // Alert colors:
   Alert: {
     ALERT_GREEN: "#1DA978",
```

Comment:
Is this a situation where we would eventually want to remove the old color? I guess I'm wondering if we should just replace the value for `ALERT_GREEN` and `ALERT_GREEN_SHADE_1/2`, although admittedly that could be a bigger change.

Reply:
Great point. I believe so, but let me look into that some more.

Reply:
I wonder if it would be better to skip introducing new color variables here, and just define them locally in the toast less file. Then we worry about adding new color variables when we have more time to devote to overhauling dewey.

Reply:
I think I'll define them locally for now.